src/main.py

```diff
@@ -183,10 +183,20 @@ async def help(interaction: Interaction):
 # Ignore non-slash commands
 @client.event
 async def on_message(message: Message):
-    # Ignore DMs by checking if the message was sent in a server
-    if message.guild is None:
+    # Ignore DMs and bot's own messages
+    if message.guild is None or message.author.bot:
         return
 
+    # Count user's messages in channel
+    count = 0
+    async for msg in message.channel.history(limit=None):
+        if msg.author.id == message.author.id:
+            count += 1
+
+    # If the user has sent less than 3 messages in the channel, check for spam
+    if count < 3:
```

Comment:
Can we extract this out into a constant that sits at the top of the file? Maybe something like `SPAM_CHECK_MIN_MSG = 3` or something like that. Relatively descriptive but not too long either.